bird-zentangle-1.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
```

Comment:
Nice use of a template on all these art detail pages. Makes for a consistent user experience.